### DISCUSSION TITLE
The export premium plugin is not enabled on your API key. Upgrade your account.

### DISCUSSION THREAD

**Discussion by kolserdav** [tinymce/tinymce-react#443] [Q&A] [answered]
I use such plugins. Everything worked for about two weeks, and today suddenly this message appeared. As I understand it, you can’t hope that in the end all the plug-ins, up to one, will sooner or later turn out to be paid!?
```
'advlist',
'advcode',
'advtable',
'autolink',
'checklist',
'export',
'lists',
'link',
'charmap',
'preview',
'anchor',
'searchreplace',
'visualblocks',
'powerpaste',
'fullscreen',
'formatpainter',
'insertdatetime',
'media',
'table',
```

**Comment by kolserdav** [comment]
I removed all plugins and the warning disappeared. It's strange that it didn't show up right away. It looks like a tricky trade trick :)

**Reply by danoaky-tiny** [reply, marked as answer]
Yes you would have been on the 2 week free trial as explained when you created an account. 
We have our premium plugins listed here: https://www.tiny.cloud/docs/tinymce/6/plugins/#premium-plugins.
